docs/modules/ROOT/pages/getting-started.adoc

```diff
@@ -26,7 +26,7 @@ cd neo4j-graphql-example
 +
 [source, bash, indent=0]
 ----
-npm init --yes
+npm init es6 --yes
```

Comment:
Shall we add a NOTE just underneath this code snippet to point out the usage of `es6`? Wdyt?

Reply:
Addressed in https://github.com/neo4j/graphql/pull/3239/commits/e0957d53f0af9d15077079d46601473a7dc52889